k9mail/src/main/java/com/fsck/k9/K9.java

```diff
@@ -233,7 +233,7 @@
     private static boolean hideSpecialAccounts = false;
     private static boolean autofitWidth;
     private static boolean quietTimeEnabled = false;
-    private static boolean notificationDuringQuietTimeEnabled = true;
+    private static boolean notificationDuringQuietTimeDisabled = true;
```

Comment:
Having a negative in the variable/method name makes the code harder to read. Also, you inverted the logic but didn't invert the default value.
I suggest sticking to the original name.